File: Introduction/gcd_start.java
// run: javac gcd_start.java && java gcd_start
public class gcd_start {

    private static int gcd(int a, int b) {
        while(b != 0) {
            int temp = a;
            a = b;
            b = temp % b;
        }
        return a;
    }

    public static void main(String[] args) {
        System.out.print(gcd(60, 96));
        System.out.println();
        System.out.print(gcd(20, 8));
        System.out.println();
    }
}

File: Introduction/gcd_start.cpp
#include <iostream>

using namespace std;

int gcd(int a, int b)
{
    while (b != 0)
    {
        int temp = a;
        a = b;
        b = temp % b;
    }
    return a;
}

int main()
{
    cout << gcd(60, 96) << endl;
    cout << gcd(20, 8) << endl;
    return 0;
}